src/scripts/main.js

```diff
@@ -5,4 +5,47 @@ const people = require('./lib/people');
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
+const table = document.querySelector('.dashboard > tbody');
+
 // write your code here
+for (let i = 0; i < 6; i++) {
+  const row = document.createElement('tr');
+  const rowItemName = document.createElement('td');
+
+  rowItemName.textContent = people[i].name;
+
+  const rowItemGender = document.createElement('td');
+
+  if (people[i].sex === 'm') {
+    rowItemGender.textContent = 'Male';
```

Comment:
Consider using a ternary operator to make the gender assignment more concise. Example: `rowItemGender.textContent = people[i].sex === 'm' ? 'Male' : 'Female';`